src/ggshield-webview/gitguardian-quota-webview.ts

```diff
@@ -58,7 +58,7 @@ export class GitGuardianQuotaWebviewProvider
           <p>Loading...</p>
         </body>
         </html>`;
-    } else if (this.quota !== undefined) {
+    } else if (this.quota !== 0 && this.isAuthenticated) {
```

Comment:
I think we shouldn't do this here. The problem is that the quota can be 0 if the person is not an admin. Maybe we should only display this window for an authenticated admin ?

Reply:
the quota can never be undefined actually so it was useless